package.json

```
@@ -9,6 +9,7 @@
 	],
 	"scripts": {
 		"dev": "yarn --cwd ./demo install --frozen-lockfile && yarn install --frozen-lockfile",
+		"greenkeeperDev": "yarn --cwd ./demo install && yarn install",
```

Generally, we feed kebab-case data to scripts and programs we write. For consistency, then, let's re-stylize this as:

```suggestion
		"greenkeeper-dev": "yarn --cwd ./demo install && yarn install",
```

This will need updating in the `.travis.yml` file, too.